### PR TITLE
Support of helm 3

### PR DESCRIPTION
**Pull request by jonasrutishauser**
Full support of helm v3.0.0-beta.3 and helm 2.

I implemented this independent of #149 but it will be solution for #147 which supports all commands.

### CODE REVIEW

File: cmd/upgrade.go [hunk 1]
```diff
@@ -116,10 +119,16 @@ func (d *diffCmd) run() error {
 		return err
 	}
 
-	releaseResponse, err := d.client.ReleaseContent(d.release)
+	releaseResponse, err := d.deployedRelease(d.release)
+	var notDeployed string
+	if isHelm3() {
+		notDeployed = fmt.Sprintf("%q has no deployed releases", d.release)
```
Comment (mumoshu) [inline cmd/upgrade.go]: Ah good catch! I missed this in my PR #149

**Comment by mumoshu** [comment]
@jonasrutishauser Thanks for your contribution! This looks so nice 💯 

I'm now trying to build and test this myself, but had no luck so far.

`glide install && HELM_HOME=$HOME/Library/helm make install` emits a bunch of compilation errors perhaps due to inconsistent dependencies https://gist.github.com/mumoshu/1dfda422fce533493128fa93f07ef9ac

Do you have any clue on this?

**Comment by jonasrutishauser** [comment]
I had many difficulties with glide too and after a `glide up` the build did not work, but after a `make bootstrap` which calls `glide install --strip-vendor` the build worked.
I even used more time with glide than with the implementation.

**Comment by databus23** [comment]
@jonasrutishauser Thanks for the PR. I think we need to make a general decision how this plugin should handle the rendering part for helm3 as there are some potential compatibility issues across versions of helm that I'm worried about. I'm glad to here your thought on this comment: https://github.com/databus23/helm-diff/pull/149#issuecomment-529403225

**Comment by databus23** [comment]
Closed in favour of #151

**Pull request closed without merging**